app/master/cluster_runner_config.py

```diff
@@ -1,23 +1,13 @@
-import sys
```

Comment:
The majority of this change is copy/pasting the contents of cluster_runner_config.py to job_config.py.